### ISSUE TITLE
[1.8.0] iPhone 6 / 9.2.1 - Sort "My Music” results - Only works once

### ISSUE THREAD

**Issue by xiuie** [Lomotif/lomotif-ios-issues#164]
Frequency: Every time

Action:
1. Launch Lomotif
2. Tap on START button
3. Select Camera Roll on Albums screen
4. Select few photos from Camera Roll
5. Go to My Music tab
6. Tap on Hamburger
7. Choose one option e.g. “Playlist"
8. Results sort as expected
9. Choose hamburger
10. Tapping doesn’t open the menu again

Expected result: Hamburger menu to display options whenever tapped

Actual result: Hamburger menu works only once

**Comment by xiuie** [comment]
Duplicate of #160 

My mistake, closing
